Source/EasyNetQ/ConnectionConfiguration.cs

```diff
@@ -74,13 +76,8 @@ public ConnectionConfiguration()
 
         private void SetDefaultClientProperties(IDictionary<string, object> clientProperties)
         {
-            string applicationNameAndPath = null;
-#if !NETFX
-            var version = GetType().GetTypeInfo().Assembly.GetName().Version.ToString();
-#else
-            var version = Assembly.GetExecutingAssembly().GetName().Version.ToString();
-#endif
-            applicationNameAndPath = Environment.GetCommandLineArgs()[0];
+            var version = typeof(ConnectionConfiguration).Assembly.GetName().Version.ToString();
```

Comment:
Changed to `typeof(ConnectionConfiguration)` since in theory `GetType()` **may** return inherited class from other assembly, not EasyNetQ.

Reply:
not actual naymore